docs/equinix/index.md

```diff
@@ -1,2 +1,171 @@
-# Equinix Metal Cookbooks
-### Recipes for using and deploying DLTs using Sextant on Equinix Metal
\ No newline at end of file
+# Equinix Metal Cookbook
+
+Recipe for using and deploying blockchain networks using Sextant on Equinix Metal
+
+## Setting up for Equinix Metal
+
+* [Log into Equinix Metal Portal](https://console.equinix.com/)
+* Under user profile set up "Personal SSH Keys" for server access
+* [Optionally set up Personal API Keys](https://metal.equinix.com/developers/api/)
+* Create "New Project" (we used one Project to deploy all servers in all locations)
+
+## Provisioning Equinix Servers
+
+* Select Servers/On Demand
+* Choose a location (Amsterdam, Chicago, Dallas, etc.)
+* Choose a Server Type (we used `c3.small.x86`)
+* Choose an Operating System (we tested RKE2 on `Ubuntu 18.04LTS` and `Ubuntu 20.04LTS`)
```

Comment:
Do we have a recommendation?

Reply:
Actually, we have no recommendation - it works fine on any platform we tried (including CentOS/RHEL).

Reply:
I've added link to RKE2 docs that has all supported version listed.